CWE/decisions/claim_greater.txt

```diff
@@ -5521,8 +5513,8 @@ NOT = { government = democracy government = hms_government }
 		effect = {
 			badboy = 12
 			set_country_flag = claim_greater_arabia
-			any_country = { 
-				any_owned = { limit = { pop_majority_culture = arab NOT = { is_core = XAL } } add_core = XAL } 
+			any_country = {
+				any_owned = { limit = { NOT = { is_core = XAL } NOT = { is_core = PER } NOT = { is_core = ARM } NOT = { is_core = AZB } NOT = { is_core = GEO } NOT = { is_core = TUR } OR = { continent = mena } } add_core = XAL } 
```

Comment:
all_core = { limit = { NOT = { is_core = XAL } NOT = { is_core = PER } NOT = { is_core = ARM } NOT = { is_core = AZB } NOT = { is_core = GEO } NOT = { is_core = TUR } continent = mena } add_core = XAL }